src/engine/specification.md

```diff
@@ -310,3 +322,19 @@ The payload build process is specified as follows:
 1. The call **MUST** return `-32001: Unknown payload` error if the build process identified by the `payloadId` does not exist.
 
 1. Client software **MAY** stop the corresponding build process after serving this call.
+
+### engine_getTransitionConfigurationV1
+
+#### Request
+
+* method: `engine_getTransitionConfigurationV1`
+* params: *empty list*
```

Comment:
The alternative is that it's more of a ping/pong where each send their configuration. Thus both can see a mismatch and both can surface an error.

Do you think this would help reduce chance of user failures by both logs having the error?

Reply:
+1 to this idea, basically `params` is the CL's configuration and `response` is the EL's, and both sides SHOULD log a warning if the other configuration doesn't match.

Reply:
I like this idea! 👍 I think we may re-use the same data structure. When `TERMINAL_BLOCK_HASH` is set CL client will send `terminalBlockNumber: 0` as a block number is not a part of CL's configuration.

Reply:
Although, what if it's called not by CL but by a script that checks every EL client in a cluster and then matches values with what are expected to be? Do we want to design for this use case? If yes, then we should make the parameter optional.

Reply:
A cluster/advanced test could also add the config as a param. I think it's not worth the complxity of making this optional